data/korath/korath weapons.txt

```diff
@@ -576,3 +576,116 @@ effect "slicer impact"
 	"random spin" 2
 	"random velocity" .1
 	"velocity scale" -.002
+
+outfit "Firelight Storage Rack"
+	category "Ammunition"
+	cost 27400
+	thumbnail "outfit/firelight rack"
+	"mass" 3.3
+	"outfit space" -6
+	"firelight missile capacity" 9
+	ammo "Firelight Missile"
+	description "This rack stores additional Firelight missiles to feed into a Firelight Bank."
+
+outfit "Firelight Missile"
+	category "Ammunition"
+	cost 7600
+	thumbnail "outfit/firelight"
+	"mass" 0.3
+	"firelight missile capacity" -1
+	"energy generation" 0.02
+	"heat generation" 0.1
+	description `The reactor element inside a Firelight missile must be kept minimally active in order to ensure it is ready for launch.  Although terribly inefficient, it nonetheless produces a trickle of power that can be tapped into when properly installed.`
```

Comment:
```suggestion
	description `The reactor element inside a Firelight missile must be kept minimally active in order to ensure it is ready for launch. Although terribly inefficient, it nonetheless produces a trickle of power that can be tapped into when properly installed.`
```
Just noticed this real quick looking through descriptions.

Reply:
Good catch. Thank you